### ISSUE TITLE
val loss gets invoked when vx1=None

### ISSUE THREAD

**Issue by prtsk** [Michaelvll/DeepCCA#3]

**Comment by prtsk** [comment]
### Line 93 in main.py include val_loss irrespective of vx1, vx2=None. This needs to be handled for vx1, vx2 = None. I have fixed it for myself, but thought you should know about it.

**Comment by Michaelvll** [comment]
I fixed it in https://github.com/Michaelvll/DeepCCA/commit/e9775bb896313d3d75588edfb5941107c02eb3f9
Thank you!
